statsmodels/sandbox/distributions/mv_normal.py

```diff
@@ -926,11 +926,10 @@ def affine_transformed(self, shift, scale_matrix):
 
         Returns
         -------
-        mvt : instance of MVT
-            instance of multivariate t distribution given by affine
+        mvt : instance of MVNormal
```

Comment:
mvt -> mvn

Reply:
Sounds good.  BTW while looking through the sandbox.distributions code I found a lot of it to be really nice (also TheilGLS stood out).  I'm looking forward to seeing what else you have in mind in this area.